src/main/java/FlyChat/Tasks/Task.java

```diff
@@ -0,0 +1,48 @@
+package FlyChat.Tasks;
+
+/**
+ * Represents the Tasks of the user.
+ */
+public class Task {
```

Comment:
Perhaps this class should be an Abstract task?
```suggestion
public abstract class Task {
```